test/common/constants.ts

```diff
@@ -10,7 +10,7 @@ export const TEST_POSITION_ID3 = '0xe85f118865dfdae84a450360d6e0c3e601218f312a0a
 export const TEST_POSITION_ID4 = '0xa1f496b9eecc3e711a3cc741ba3d34abfdebdb73e6fb3ff5ef187196350177dc';
 export const TEST_OPEN_POSITION = '0xa9a3ba329fce666f22682ad894a0031c2cb44d62451a1a52b67df0857175b547';
 
-export const TEST_USER_ID1 = '0x850cf0ff7b0bf01255ddfce1d3dc4e1cbfeaf5af';
+export const TEST_USER_ID1 = 'PERP-170141183460469231731687303715884105729';
```

Comment:
User ID is incorrect? Is this just for testing purposes?

Reply:
yes

Reply:
it's the perp_Account ID